instrumentation/grpctrace/interceptor_test.go

```diff
@@ -97,7 +97,9 @@ func TestUnaryClientInterceptor(t *testing.T) {
 		{
 			name: "/github.com.serviceName/bar",
 			expectedAttr: map[kv.Key]value.Value{
-				standard.RPCServiceKey:  value.String("serviceName"),
+				standard.RPCSystemKey:   value.String("grpc"),
+				standard.RPCServiceKey:  value.String("/github.com.serviceName/bar"),
```

Comment:
I'm not sure that the service name should include the method.  From [the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md#span-name) I'd expect `RPCServiceKey` to contain `$package.$service` and not `$method`.